Source/WebKit/Configurations/BaseTarget.xcconfig

```diff
@@ -31,8 +31,12 @@ SYSTEM_FRAMEWORK_SEARCH_PATHS = $(inherited) $(WK_PRIVATE_SDK_DIR)$(SYSTEM_LIBRA
 WK_FORMATREADERBUNDLE_CODESIGNING_DEFINES = ENABLE_LOWER_FORMATREADERBUNDLE_CODESIGNING_REQUIREMENTS;
 WK_FORMATREADERBUNDLE_CODESIGNING_DEFINES[config=Production] = ;
 
+WK_BUNDLE_VERSION_DEFINES = WEBKIT_BUNDLE_VERSION='"$(BUNDLE_VERSION)"';
```

Comment:
nit: Does the quoting here accomplish anything? It usually doesn't — nothing in the build system is doing shell expansion.

Reply:
It's definitely required due to the way `-D` works and the way `GCC_PREPROCESSOR_DEFINITIONS` works with it. You need to convince Xcode to pass the extra double quotes to `-D` for this to actually be defined as a string literal.

Consider:

```
$ cat foo.c
#include <stdio.h>

int main() {
  printf("%s\n", FOO);
}
```

Without double quotes:

```
$ clang '-DFOO=1.2.34+' -o foo foo.c
foo.c:4:18: error: invalid suffix '.34' on floating constant
  printf("%s\n", FOO);
                 ^
<command line>:1:16: note: expanded from macro 'FOO'
#define FOO 1.2.34+
               ^
foo.c:4:21: error: expected expression
  printf("%s\n", FOO);
                    ^
2 errors generated.
```

With double quotes:

```
$ clang '-DFOO="1.2.34+"' -o foo foo.c
<no error>
```

Reply:
I see, it's the _preprocessor_ that needs to see those quotes (so that it parses as a string literal). Thanks for clarifying!